CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Next Release
 ### Internal
+- [In Progress: 20 Jan 2021] Material Theme-ing Migration
+- [In Progress: 08 Feb 2021] Migrate app to use ViewBinding
+
+### Changes
+- Change "Preferred facility" to "Assigned facility"
```

Comment:
Why did we move these `Changes` and `Fixes` to next release as well? As far as I remember they were merged in into the master branch, right?

Reply:
Demo release happened on Monday, I forgot to raise a changelog PR that day. So these did not make it into the release branch.